backend/langate/network/models.py

```diff
@@ -53,28 +54,31 @@ class DeviceManager(models.Manager):
     """
 
     @staticmethod
-    def create_device(mac, name, whitelisted=False, mark=None):
+    def create_device(mac, name, whitelisted=False, bypass=False, mark=None):
         """
         Create a device with the given mac address and name
         """
         if not name:
             name = generate_dev_name()
         if not mark:
             mark = SETTINGS["marks"][0]["value"]
-
+        if whitelisted:
+            # Whitelisted devices have bypass by default
+            bypass = True
```

Comment:
bypass should be False by default for whitelisted devices

Reply:
Correct, i need to add a way to edit that and have it false by default (I forgor that we whitelist player devices too)

Reply:
Done